### ISSUE TITLE
Inconsistency in weeks values

### ISSUE THREAD

**Issue by becausealice2** [guoguo12/billboard-charts#50]
Looking trough data scraped from 2018, I am getting 604 entries at 0 weeks on the chart, 4 at 1 week, and 393 at 2 weeks.
Incidentally, the 4 entries at week 1 are [the four songs that debuted at number 1 in 2018](https://www.billboard.com/photos/426225/billboard-hot-100-number-1-song-debuts) on their debut weeks, which could be connected to issue #49 :
```
                          title  weeks
date       rank                                
2018-11-17 1       Thank U, Next      1
2018-05-19 1     This Is America      1
2018-04-21 1       Nice For What      1
2018-02-03 1          God's Plan      1
```
I think the discrepancy is because you are assigning the first week on the chart to 0 at line 286 in Billboard.py when `getPositionRowValue(rowName)` is passed `_WEEKS_ON_CHART_FORMAT`, but Billboard starts counting weeks at 1, which you can see for [This Is America on it's debut week](https://www.billboard.com/charts/hot-100/2018-05-19), but is not included for songs debuting at lower ranks, as is shown for Watch's number 16 debut that same week.

**Comment by jameswenzel** [comment]
Thanks for pointing this out! This along with #49 is indeed another case of a bad default. I've submitted a PR to fix this, and hopefully @guoguo12 can release a new version sooner rather than later.
In the meantime, though, the behavior you've identified should be consistent enough that you can add some helper logic to smooth over these rough spots, if this is affecting your work. 
Sorry for the dirty data!

**Comment by becausealice2** [comment]
Cleaning it up was no problem at all... I'm just not confident enough in my Python skills to contribute directly to the repository. So, thanks :)

**Comment by guoguo12** [comment]
Fixed in 5.0.1. @becausealice2, thanks for reporting this (and #49)! These are really important edge cases!

**Comment by guoguo12** [comment]
Alright, I went through all of the songs from 2018 and got the following distribution of ranks:

````
Counter({1: 599, 2: 389, 3: 332, 4: 291, 5: 270, 6: 249, 7: 229, 8: 216, 9: 203, 10: 191, 11: 182, 12: 175, 13: 166, 14: 160, 15: 152, 16: 145, 17: 145, 18: 143, 19: 135, 20: 125, 21: 78, 22: 67, 23: 59, 24: 52, 25: 48, 26: 45, 27: 42, 28: 38, 29: 36, 30: 29, 31: 26, 32: 25, 34: 25, 33: 24, 35: 24, 36: 21, 37: 19, 38: 17, 39: 15, 40: 14, 41: 11, 42: 10, 43: 8, 44: 8, 50: 7, 51: 7, 52: 7, 45: 6, 46: 6, 49: 6, 47: 5, 48: 5, 54: 3, 53: 2, 55: 2, 56: 2, 57: 2, 58: 1, 59: 1})
````

This seems more correct.

Source code:

````python
from collections import Counter

import billboard

CHART = 'hot-100'

counter = Counter()
chart = billboard.ChartData(CHART, date='2018-12-29')
while chart.date > '2018':
    print(chart.date)
    for track in chart:
        counter[track.weeks] += 1
    chart = billboard.ChartData(CHART, date=chart.previousDate)
print(counter)
````
